tests/unit/fixed-toolbar/fixedToolbar.js

```diff
@@ -2,45 +2,55 @@
  * mobile Fixed Toolbar unit tests
  */
 (function($){
-	module('jquery.mobile.toolbar.js');
+	module( "jquery.mobile.toolbar.js" );
 
-	test( "Fixed Header Structural Classes are applied correctly", function(){
+	test( "Fixed Header Structural Classes are applied correctly", function( assert ){
 		//footer
-		ok( !$('#classes-test-a').hasClass('ui-header-fixed'), 'An ordinary header should not have fixed classes');
-		ok( $('#classes-test-b').hasClass('ui-header-fixed'), 'An header with data-position=fixed should have ui-header-fixed class');
-		ok( $('#classes-test-c').hasClass('ui-header-fullscreen'), 'An header with data-position=fixed and data-fullscreen should have ui-header-fullscreen class');
+		assert.lacksClasses( $( "#classes-test-a" ), "ui-toolbar-header-fixed", 
+			"An ordinary header should not have fixed classes" );
+		assert.hasClasses( $( "#classes-test-b" ), "ui-toolbar-header-fixed", 
+			"A header with data-position=fixed should have ui-toolbar-header-fixed class" );
+		assert.hasClasses( $( "#classes-test-c" ), "ui-toolbar-header-fullscreen",
+			"A header with =fixed and data-fullscreen gets ui-toolbar-header-fullscreen class" );
 
 		//footer
-		ok( !$('#classes-test-d').hasClass('ui-footer-fixed'), 'An ordinary footer should not have fixed classes');
-		ok( $('#classes-test-e').hasClass('ui-footer-fixed'), 'A footer with data-position=fixed should have ui-footer-fixed class"');
-		ok( $('#classes-test-f').hasClass('ui-footer-fullscreen'), 'A footer with data-position=fixed and data-fullscreen should have ui-footer-fullscreen class');
+		assert.lacksClasses( $( "#classes-test-d" ), "ui-toolbar-footer-fixed", 
+			"An ordinary footer should not have fixed classes");
+		assert.hasClasses( $( "#classes-test-e" ), "ui-toolbar-footer-fixed", 
+			"A footer with data-position=fixed should have ui-toolbar-footer-fixed class" );
+		assert.hasClasses( $( "#classes-test-f" ), "ui-toolbar-footer-fullscreen", 
+			"A footer with =fixed and data-fullscreen gets ui-toolbar-footer-fullscreen class" );
 
 		//parent
-		ok( $('#classes-test-b').closest( ".ui-page" ).hasClass( "ui-page-header-fixed" ), "Parent page of a fixed header has class ui-page-header-fixed" );
-		ok( $('#classes-test-e').closest( ".ui-page" ).hasClass( "ui-page-footer-fixed" ), "Parent page of a fixed footer has class ui-page-header-fixed" );
+		assert.hasClasses( $( "#classes-test-b" ).closest( ".ui-page" ),
+			"ui-page-header-fixed", 
+			"Parent page of a fixed header has class ui-page-header-fixed" );
+		assert.hasClasses( $( "#classes-test-e" ).closest( ".ui-page" ),
+			"ui-page-footer-fixed", 
+			"Parent page of a fixed footer has class ui-page-header-fixed" );
```

Comment:
we should save `$( "#classes-test-b" )` and `$( "#classes-test-e" )` since they are reused